pkg/cluster/cluster_test.go

```diff
@@ -1088,3 +1090,171 @@ func TestValidUsernames(t *testing.T) {
 		}
 	}
 }
+
+func TestCluster_compareContainers(t *testing.T) {
```

Comment:
Could you reduce the tests to just `TestComparePorts` cases?

Reply:
Sure. Done